biit_server/meeting_handler.py

```diff
@@ -749,6 +754,9 @@ def matchup(request, auth):
         match = [min_potential_id, min_user_id]
 
         random_id = str(uuid.uuid4())
+
+        zoom_info = create_meeting(in_a_week.timestamp(), 30, match)
```

Comment:
How is zoom handled when a person wants to meet up in person? Is it still generated, but not sent?